R/wwinference.R

```diff
@@ -160,6 +163,18 @@ wwinference <- function(ww_data,
     )
   }
 
+  fit_opts_use <- get_mcmc_options() # get defaults
+  # this overwrites defaults with all and only the values the user sets in
+  # `fit_opts`
+  fit_opts_use[names(fit_opts)] <- fit_opts
+
+  # Check that the fit options passed to wwinference are valid cmdstanr::sample
+  # arguments
+  checkmate::assert_names(names(fit_opts),
```

Comment:
add a test to `test_wwinference.R` to confirm this error gets raised when it should?